pkg/serviceaccount/serviceaccounts_controller.go

```diff
@@ -44,8 +43,8 @@ func nameIndexFunc(obj interface{}) (string, error) {
 
 // ServiceAccountsControllerOptions contains options for running a ServiceAccountsController
 type ServiceAccountsControllerOptions struct {
-	// Names is the set of service account names to ensure exist in every namespace
-	Names util.StringSet
+	// ServiceAccounts is the list of service accounts to ensure exist in every namespace
+	ServiceAccounts []api.ServiceAccount
```

Comment:
> Confused. Since ServiceAccounts reference a Secret object, in a specific namespace, you are expecting that those are created before this namespace is created?

This field (previously `Names`) is a useful field for extenders who want to ensure that some service accounts always exist in the namespace.  These service accounts aren't merely created, they constantly ensured to be present.  However, there was no way to label or annotate the service accounts that were getting created by this controller.  Changing `Names` to be a full `api.ServiceAccount` struct allows particular labels and annotations to be applied.